opm/parser/eclipse/Units/UnitSystem.cpp

```diff
@@ -129,6 +138,9 @@ namespace {
         1, /* gas-oil ratio */
```

Comment:
> Are there many ratios in inputs?

Formation volume factors are input in PVDO, PVDG, PVTO, PVTG, PVTW etc. (pvt data).